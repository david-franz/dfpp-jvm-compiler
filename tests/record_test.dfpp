module tests.record

fn main() {
  let rec = { id = 123, title = "df++" }
  print("id=" + rec.id)
  print("title=" + rec.title)
}
