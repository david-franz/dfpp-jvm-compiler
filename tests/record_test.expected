id=123
title=df++
